tests/logic.rs

```diff
@@ -237,7 +237,10 @@ fn append_logic_xor() {
     // Compile common circuit descriptions for the prover and verifier to be
     // used by all tests
     let label = b"append_logic_xor";
-    let mut rng = StdRng::seed_from_u64(0xdea1);
+
+    // FIXME: One test used to fail when using "0xdea1" as randomness here. It
+    // needs to be tackcled soon or later.
+    let mut rng = StdRng::seed_from_u64(0xded1);
```

Comment:
If just mentioned in a `FIXME` somewhere in the tests, I'm afraid that the fix to this might get forgotten.

I would rather have it that you leave the seed as it was, isolate the failing testcase and create a bug-report issue. Then you can ignore the isolated failing test with a reference to the issue that would solve it.